src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/BorderTests/Border_CornerRadius.xaml

```diff
@@ -1,19 +1,19 @@
 ﻿<Page x:Class="Uno.UI.Samples.UITests.BorderTestsControl.Border_CornerRadius"
-      xmlns:controls="using:Uno.UI.Samples.Controls"
-      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-      xmlns:local="using:Uno.UI.Samples.UITests.BorderTestsControl"
-      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-      xmlns:ios="http://uno.ui/ios"
-      xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-      xmlns:android="http://uno.ui/android"
-      mc:Ignorable="d ios android">
-
-    <controls:SampleControl SampleDescription="Border_Simple">
-        <controls:SampleControl.SampleContent>
-            <DataTemplate>
-                <StackPanel>
+	  xmlns:controls="using:Uno.UI.Samples.Controls"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.UI.Samples.UITests.BorderTestsControl"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:ios="http://uno.ui/ios"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:android="http://uno.ui/android"
+	  mc:Ignorable="d ios android">
+
+	<controls:SampleControl SampleDescription="Border_Simple">
+		<controls:SampleControl.SampleContent>
+			<DataTemplate>
```

Comment:
You can remove all those `<SampleControl />`/`<DataTemplate />` lines and go directly to the content of your sample.